packages/health/example/lib/main.dart

```diff
@@ -19,71 +17,49 @@ enum AppState {
   FETCHING_DATA,
   DATA_READY,
   NO_DATA,
+  AUTHORIZED,
   AUTH_NOT_GRANTED,
   DATA_ADDED,
+  DATA_DELETED,
   DATA_NOT_ADDED,
+  DATA_NOT_DELETED,
   STEPS_READY,
 }
 
 class _HealthAppState extends State<HealthApp> {
   List<HealthDataPoint> _healthDataList = [];
   AppState _state = AppState.DATA_NOT_FETCHED;
-  int _nofSteps = 10;
-  double _mgdl = 10.0;
+  int _nofSteps = 0;
+
+  // Define the types to get.
+  // NOTE: These are only the ones supported on Androids new API Health Connect.
+  // Both Android's Google Fit and iOS' HealthKit have more types that we support in the enum list [HealthDataType]
+  // Add more - like AUDIOGRAM, HEADACHE_SEVERE etc. to try them.
+  static final types = dataTypesAndroid;
+
+  // Or selected types
```

Comment:
Do we need to keep all this commented code?

Reply:
It is in example app, not in the plugin. It can be there to easier debugging purposes and doesn't affect the functionality.